common/workunit/workunit.cpp

```diff
@@ -2487,6 +2487,39 @@ class CompoundAggregator : implements StatisticAggregator
     IArrayOf<StatisticAggregator> aggregators;
 };
 
+class CostAggregator : public StatisticAggregator
+{
+public:
+    CostAggregator(StatsAggregation & _summary) : StatisticAggregator(StCostExecute), summary(_summary) {}
+
+    virtual void noteStatistic(StatisticKind kind, unsigned __int64 value, IConstWUStatistic & extra) override
+    {
+        summary.noteValue(value);
+    }
+
+protected:
+    StatsAggregation & summary;
+};
+
+extern WORKUNIT_API __int64 aggregateCost(IConstWorkUnit * wu)
```

Comment:
minor: we do not normally declare the implementation with extern <..>. Normally we only declare extern in the header.
I know there are few others like this in this file, but majority elsewhere are only in header.